config/vufind/BlenderMappings.yaml

```diff
@@ -0,0 +1,292 @@
+    #        media: "1/Other/Electronic/"
+    #        newspaper_articles: "1/Journal/NewspaperArticle/"
+    #        other: "1/Other/Other/"
+    #        patents: "1/OtherText/Text/"
+    #        reference_entrys: "1/OtherText/Text/"
+    #        reports: "1/OtherText/ResearchReport/"
+    #        research_datasets: "1/Other/ResearchData/"
+    #        reviews: "1/OtherText/Text/"
+    #        scores: "1/MusicalScore/MusicalScore/"
+    #        statistical_data_sets: "1/Other/Other/"
+    #        technical_reports: "1/OtherText/Text/"
+    #        text_resources: "1/OtherText/Text/"
+    #        websites: "1/Other/Electronic/"
+    #        web_resources: "1/Other/Electronic/"
+
+    author_facet:
+      Mappings:
+        Solr:
+          Field: author_facet
+        Primo:
+          Field: creator
+        EDS:
+          Field: "LIMIT|AU"
+    topic:
+      Mappings:
+        Solr:
+          Field: topic
+        Primo:
+          Field: topic
+        EDS:
+          Field: SubjectEDS
+    publish_date:
+      Mappings:
+        Solr:
+          Field: publish_date
+        Primo:
+          Field: creationdate
+        EDS:
+          Field: PublicationDate
+    language:
+      Mappings:
+        Solr:
+          Field: language
+        Primo:
+          Field: lang
+    fulltext:
+      Type: boolean
+      Mappings:
+        Solr:
+          Field: fulltext_boolean
+        Primo:
+          Field: pcAvailability
+          Values:
+            "false": 1
+          DefaultValue: "true"
+        EDS:
+          Field: "LIMIT|FT"
+          Values:
+            "y": true
+    online:
+      Type: boolean
+      Mappings:
+        Solr:
+          Field: online_boolean
+        Primo:
+          Field: tlevel
+          Values:
+            online_resources: true
+    open_access:
+      Type: boolean
+      Mappings:
+        Solr:
+          Field: open_access_boolean
+        Primo:
+          Field: tlevel
+          Values:
+            open_access: true
+    peer_reviewed:
+      Type: boolean
+      Mappings:
+        Solr:
+          Field: peer_reviewed_boolean
+        Primo:
+          Field: tlevel
+          Values:
+            peer_reviewed: true
+        EDS:
+          Field: "LIMIT|RV"
+          Values:
+            "y": true
+# Search type mappings between backends
+Search:
+  Fields:
+    AllFields:
+      Mappings:
+        Solr: AllFields
+        Primo: AllFields
+        EDS: AllFields
+    Title:
+      Mappings:
+        Solr: Title
+        Primo: Title
+        EDS: TI
+    Author:
+      Mappings:
+        Solr: Author
+        Primo: Author
+        EDS: AU
+    Subject:
+      Mappings:
+        Solr: Subject
+        Primo: Subject
+        EDS: SU
+# Sort mappings between backends
+Sorting:
+  Fields:
+    relevance:
+      Mappings:
+        Solr: relevance
+        Primo: relevance
+        EDS: relevance
+    year:
+      Mappings:
+        Solr: year # This is special for Solr
```

Comment:
Not sure what this "special" comment means. Are you referring to the fact that VuFind remaps this internally? If so, is it special for VuFind rather than special for Solr? Is the comment actually relevant?